### ISSUE TITLE
Cart component not rendering when clicked from the information section

### ISSUE THREAD

**Issue by Akash1965** [arpittyagirocks/Humari-Dukan#26]
Cart component is not rendering on clicking

https://github.com/arpittyagirocks/Humari-Dukan/assets/115504480/b3ce5b83-a8d1-429d-b747-18602de8b13a

**Comment by arpittyagirocks** [comment]
Hey, @Akash1965, Thank you for pointing out the bug

**Comment by anjali29singh** [comment]
@arpittyagirocks  there is no cart.js file therefore it is not rendering

**Comment by arpittyagirocks** [comment]
@anjali29singh Yes you are right, i have removed the cart.js file, it's no longer there

**Comment by anjali29singh** [comment]
can you tell want do you want to render on clicking the Cart option

**Comment by arpittyagirocks** [comment]
@anjali29singh, I am currently not planning to Add anything for this option, As you want to contribute, you may add a sorting feature to it.
- Users should be able to sort the products visible by price and rating.
- If you look at how searching feature is working, it wont be hard. 
- Also create an issue for it

Ask me if you have any doubt.
